docs/quickstarts/kvm-libvirt.md

```diff
@@ -0,0 +1,284 @@
+# Lokomotive KVM libvirt quickstart guide
+
+## Contents
+
+* [Introduction](#introduction)
+* [Requirements](#requirements)
+* [Step 1: Install lokoctl](#step-1-install-lokoctl)
+* [Step 2: Prepare the VM Image](#step-2-prepare-the-vm-image)
+* [Step 3: libvirtd Setup](#step-3-libvirtd-setup)
+* [Step 4: User Setup](#step-4-user-setup)
+* [Step 5: Create a cluster configuration](#step-5-create-a-cluster-configuration)
+* [Step 6: Check ssh-agent](#step-6-check-ssh-agent)
+* [Step 7: Deploy the cluster](#step-7-deploy-the-cluster)
+* [Verification](#verification)
+* [Using the cluster](#using-the-cluster)
+* [Cleanup](#cleanup)
+* [Troubleshooting](#troubleshooting)
+
+## Introduction
+
+This guide shows how to create a Lokomotive cluster with Flatcar Container Linux VMs on your local machine.
+By the end of this guide, you'll have a basic Lokomotive cluster running that consist of controller and worker nodes.
+
+Controllers are provisioned to run an `etcd-member` peer and a `kubelet` service.
+Workers run just a `kubelet` service. A one-time [bootkube](https://github.com/kubernetes-incubator/bootkube)
+bootstrap process schedules the `apiserver`, `scheduler`, `controller-manager`, and `coredns` pods on the controllers
+and schedules `kube-proxy` and `calico` on every node.
+A generated `kubeconfig` provides `kubectl` access to the cluster.
+
+## Requirements
+
+* A Linux host OS.
+* At least 4 GB of available RAM.
+* An SSH key pair.
+* Terraform `v0.12.x`
+  [installed](https://learn.hashicorp.com/terraform/getting-started/install.html#install-terraform).
+* [terraform-provider-ct](https://github.com/poseidon/terraform-provider-ct),
```

Comment:
No need for this, it gets automatically installed now

Reply:
#1078 :smile: